src/script/widgets/WMSStylesDialog.js

```diff
@@ -49,6 +64,10 @@ gxp.WMSStylesDialog = Ext.extend(Ext.Container, {
      addStyleTip: "Add a new style",
     /** api: config[chooseStyleText] (i18n) */    
     chooseStyleText: "Choose style",
+    /** api: config[classifyStyleText] (i18n) */
+    classifyStyleText:"Classify",
```

Comment:
Done (noticed that WMSStylesDialog section was missing for French file; added it in)